apps/node/src/app/main/events/user_related.py

```diff
@@ -0,0 +1,387 @@
+    except (RoleNotFoundError, UserNotFoundError) as e:
+        response_body[RESPONSE_MSG.ERROR] = str(e)
+        logging.warning("User not found in post-role", exc_info=e)
+    except (TypeError, MissingRequestKeyError, PyGridError, JSONDecodeError) as e:
+        response_body[RESPONSE_MSG.ERROR] = str(e)
+    except Exception as e:
+        response_body[RESPONSE_MSG.ERROR] = str(e)
+
+    return dumps(response_body)
+
+
+@body_token_required
+def get_all_users_socket(current_user: User, message: dict) -> str:
+    response_body = {}
+
+    try:
+        private_key = message.get("private-key")
+        if private_key is None:
+            raise MissingRequestKeyError
+
+        if private_key != current_user.private_key:
+            raise InvalidCredentialsError
+
+        users = get_all_users(current_user, private_key)
+        users = [expand_user_object(user) for user in users]
+        response_body = {RESPONSE_MSG.SUCCESS: True, "users": users}
+
+    except (InvalidCredentialsError, AuthorizationError) as e:
+        response_body[RESPONSE_MSG.ERROR] = str(e)
+        logging.warning("User credentials are invalid", exc_info=e)
+    except (RoleNotFoundError, UserNotFoundError) as e:
+        response_body[RESPONSE_MSG.ERROR] = str(e)
+        logging.warning("User not found in get-roles", exc_info=e)
+    except (TypeError, MissingRequestKeyError, PyGridError, JSONDecodeError) as e:
+        response_body[RESPONSE_MSG.ERROR] = str(e)
+    except Exception as e:
+        response_body[RESPONSE_MSG.ERROR] = str(e)
+
+    return dumps(response_body)
+
+
+@body_token_required
+def get_specific_user_socket(current_user: User, message: dict) -> str:
+    response_body = {}
+
+    try:
+        user_id = message.get("user-id")
+        private_key = message.get("private-key")
+        if private_key is None:
+            raise MissingRequestKeyError
+
+        if private_key != current_user.private_key:
+            raise InvalidCredentialsError
+
+        user = get_specific_user(current_user, private_key, user_id)
+        user = expand_user_object(user)
+        response_body = {RESPONSE_MSG.SUCCESS: True, "user": user}
+
+    except (InvalidCredentialsError, AuthorizationError) as e:
+        response_body[RESPONSE_MSG.ERROR] = str(e)
+        logging.warning("User credentials are invalid", exc_info=e)
+    except (RoleNotFoundError, UserNotFoundError) as e:
+        response_body[RESPONSE_MSG.ERROR] = str(e)
+        logging.warning("User not found in get-roles", exc_info=e)
+    except (TypeError, MissingRequestKeyError, PyGridError, JSONDecodeError) as e:
+        response_body[RESPONSE_MSG.ERROR] = str(e)
+    except Exception as e:
+        response_body[RESPONSE_MSG.ERROR] = str(e)
+        logging.warning("User credentials are invalid", exc_info=e)
+
+    return dumps(response_body)
+
+
+@body_token_required
+def put_email_socket(current_user: User, message: dict) -> str:
```

Comment:
Change all of the `put_` routes to describe what it's doing. This should be something like `change_email_socket`. Also make sure to do this for all the HTTP routes.